package.json

```diff
@@ -19,7 +19,7 @@
     "find-business-days-in-range":"1.0.2"
   },
   "engines": {
-      "node": "v6.7.0",
+      "node": "v7.2",
```

Comment:
I am not fussed which. I restaged it with `>6.9.0` and it picked version 7.2, so the buildpack will pick the highest version it supports that is greater than 6.9.0.